services/frontend-v3/src/i18n/fr_CA.json

```diff
@@ -139,6 +139,7 @@
   "profile.acting.period.start.date": "Date de début intérimaire",
   "profile.address": "Adresse",
   "profile.branch": "Branche",
+  "profile.basic": "Info de base",
```

Comment:
Same comment for english... not sure why where this is used or why we would need one.

Reply:
Translation would be Information personnel or Information d'employée